x-pack/plugins/ml/public/settings/styles/main.less

```diff
@@ -7,8 +7,8 @@
 }
 
 ml-settings {
-  .management-panel {
-    .management-panel__link {
+  .mgtPanel {
+    .mgtPanel__link {
       font-size: 17px;
       line-height: 32px;
       margin-left: 6px;
```

Comment:
Should these be converted to EUI size variables?

Reply:
Oh nm. Not deleted, this is an ML one I made to match. I'll adjust.

Reply:
Blargh. What I meant to say is...

* This is an ML less file, so it doesn't have the EUI scope.
* It's being fixed separately in my ML PR.
* This fix is in here just in case this PR goes in first. Once the ML one gets in, it's replaced with Sass and properly converted.